diff --git a/dist/build/webpack/loaders/next-flight-loader/action-proxy.js b/dist/build/webpack/loaders/next-flight-loader/action-proxy.js
index ba3e71780a79676b7da2de2af851259a72f0b605..bd2f573b10062f9ce88742c622ad3fd2911b68d4 100644
--- a/dist/build/webpack/loaders/next-flight-loader/action-proxy.js
+++ b/dist/build/webpack/loaders/next-flight-loader/action-proxy.js
@@ -36,16 +36,20 @@ function createActionProxy(id, boundArgsFromClosure, action, originalAction) {
     }
     Object.defineProperties(action, {
         $$typeof: {
-            value: SERVER_REFERENCE_TAG
+            value: SERVER_REFERENCE_TAG,
+            configurable: true
         },
         $$id: {
-            value: id
+            value: id,
+            configurable: true
         },
         $$bound: {
-            value: boundArgsFromClosure
+            value: boundArgsFromClosure,
+            configurable: true
         },
         bind: {
-            value: bindImpl
+            value: bindImpl,
+            configurable: true
         }
     });
 }
diff --git a/dist/esm/build/webpack/loaders/next-flight-loader/action-proxy.js b/dist/esm/build/webpack/loaders/next-flight-loader/action-proxy.js
index 82ba2b33db5de1ee0addedf1c7644d13fd821eac..6661ffa4824ff1fc243dbe95364d9094ce04f6d0 100644
--- a/dist/esm/build/webpack/loaders/next-flight-loader/action-proxy.js
+++ b/dist/esm/build/webpack/loaders/next-flight-loader/action-proxy.js
@@ -26,16 +26,20 @@ export function createActionProxy(id, boundArgsFromClosure, action, originalActi
     }
     Object.defineProperties(action, {
         $$typeof: {
-            value: SERVER_REFERENCE_TAG
+            value: SERVER_REFERENCE_TAG,
+            configurable: true
         },
         $$id: {
-            value: id
+            value: id,
+            configurable: true
         },
         $$bound: {
-            value: boundArgsFromClosure
+            value: boundArgsFromClosure,
+            configurable: true
         },
         bind: {
-            value: bindImpl
+            value: bindImpl,
+            configurable: true
         }
     });
 }
